docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -454,18 +433,8 @@ Not intended for use with mainnet or public testnets.
 ### ottoman
 
 !!!important
-    This option is deprecated in favor of the new `--network` option.
-    It will be completely removed in the 0.9 release.
-    
-```bash tab="Syntax"
---ottoman
-```
-
-Enables accepting of blocks in an IBFT 1.0 network.  The default is `false`.
-
-!!!note
-    A Pantheon node cannot be a validator in an IBFT 1.0 network. Pantheon implements [IBFT 2.0](../Consensus-Protocols/IBFT.md).
-
+    This version is not supporting IBFT Ottoman test network anymore. 
```

Comment:
Similar to above comments. I'd just delete this option completely.  It'll be covered in the release notes that it's gone.